builtin/logical/ssh/path_config_ca.go

```diff
@@ -228,6 +251,17 @@ func (b *backend) pathConfigCAUpdate(req *logical.Request, data *framework.Field
 
 		return nil, err
 	}
+
+	if generateSigningKey {
```

Comment:
I don't care either way -- if it's not being generated you already have the public key (and have probably distributed it already too). If you feel strongly about it, then sure, change it.